collectors/python.d.plugin/spigotmc/spigotmc.chart.py

```diff
@@ -39,14 +40,26 @@
         'lines': [
             ['users', 'Users', 'absolute', 1, 1]
         ]
+    },
+    'mem': {
+        'options': [None, 'Minecraft Memory Usage', 'MiB', 'spigotmc', 'spigotmc.mem', 'line'],
+        'lines': [
+            ['mem_used', 'used', 'absolute', 1, 1],
+            ['mem_alloc', 'allocated', 'absolute', 1, 1],
+            ['mem_max', 'max', 'absolute', 1, 1]
+        ]
     }
 }
 
 _TPS_REGEX = re.compile(
+    # Examples:
+    # §6TPS from last 1m, 5m, 15m: §a*20.0, §a*20.0, §a*20.0
+    # §6Current Memory Usage: §a936/65536 mb (Max: 65536 mb)
     r'^.*: .*?'  # Message lead-in
     r'(\d{1,2}.\d+), .*?'  # 1-minute TPS value
     r'(\d{1,2}.\d+), .*?'  # 5-minute TPS value
-    r'(\d{1,2}\.\d+).*$',  # 15-minute TPS value
+    r'(\d{1,2}\.\d+).*?'  # 15-minute TPS value
+    r'(\s.*?(\d+)\/(\d+).*?: (\d+).*)?', # Current Memory Usage / Total Memory (Max Memory)
     re.X
```

Comment:
```suggestion
    re.MULTILINE
```

Test this, for me, it doesn't work w/o  re.MULTILINE

<details>
<summary>test code click click</summary>

```py
import re

_TPS_REGEX = re.compile(
    # Examples:
    # §6TPS from last 1m, 5m, 15m: §a*20.0, §a*20.0, §a*20.0
    # §6Current Memory Usage: §a936/65536 mb (Max: 65536 mb)
    r'^.*: .*?'  # Message lead-in
    r'(\d{1,2}.\d+), .*?'  # 1-minute TPS value
    r'(\d{1,2}.\d+), .*?'  # 5-minute TPS value
    r'(\d{1,2}\.\d+).*?'  # 15-minute TPS value
    r'(\s.*?(\d+)\/(\d+).*?: (\d+).*)?',  # Current Memory Usage / Total Memory (Max Memory)
    re.MULTILINE
)

data = '''§6TPS from last 1m, 5m, 15m: §a*20.0, §a*20.0, §a*20.0
§6Current Memory Usage: §a936/65536 mb (Max: 65536 mb)
'''

v = _TPS_REGEX.match(data)

print(v.groups())
```



</details>